Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -501,7 +501,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
         }
 
-        return empty($changed) ? 0 : 1;
+        return !$resolver->isDryRun() || empty($changed) ? 0 : 2;
```

Comment:
It seems not. They are actually reserved: https://msdn.microsoft.com/en-us/library/windows/desktop/ms681382%28v=vs.85%29.aspx.